src/Analyzer.Utilities/ILocationResolver.cs

```diff
@@ -6,8 +6,10 @@ namespace Microsoft.Azure.Templates.Analyzer.Utilities
     /// <summary>
     /// An interface used for resolving line numbers in a document.
     /// </summary>
-    public interface ILineNumberResolver
+    public interface ILocationResolver
     {
+        // TODO: include source file for bicep once nested templates are supported
```

Comment:
Maybe we should add a comment mentioning this in the nested templates issue/ADO story?